### PR TITLE
investing implemented

### PR DESCRIPTION
**Pull request by robrobbins**

### CODE REVIEW

File: computable/tests/conftest.py [hunk 1]
```diff
@@ -30,13 +31,13 @@ def ether_token(w3, ether_token_opts):
         abi = json.loads(f.read())
     with open(os.path.join(contract_path, 'ethertoken.bin')) as f:
         bc = f.read()
-    token = w3.eth.contract(abi=abi, bytecode=bc.rstrip('\n'))
-    tx_hash = token.constructor(w3.eth.defaultAccount,
+    t = w3.eth.contract(abi=abi, bytecode=bc.rstrip('\n'))
```
Comment (rbharath) [inline computable/tests/conftest.py:34]: Style nit, I think `token` is more pythonic than `t` here, but not a strong opinion.

File: computable/tests/conftest.py [hunk 2]
```diff
@@ -67,9 +68,9 @@ def voting(w3, market_token):
     v = w3.eth.contract(abi=abi, bytecode=bc.rstrip('\n'))
     tx_hash = v.constructor(market_token.address).transact()
     tx_rcpt = w3.eth.waitForTransactionReceipt(tx_hash)
-    voting = Voting(w3.eth.defaultAccount)
-    voting.at(w3, tx_rcpt['contractAddress'])
-    return voting
+    vg = Voting(w3.eth.defaultAccount)
```
Comment (rbharath) [inline computable/tests/conftest.py:71]: Same with `vg` versus `voting`

**Review by rbharath — APPROVED** [review]
LGTM

**Comment by robrobbins** [comment]
those are 2 different constructors tho and i didn't like overwriting the values hence token and t. I'll put something more descriptive in when i do the next contract...

**Pull request merged**